src/mygrad/nnet/losses/negative_log_likelihood.py

```diff
@@ -0,0 +1,65 @@
+import numpy as np
+
+from mygrad import Tensor, sum
```

Comment:
Nit: let's import sum as `mygrad_sum` or something like that. Just so we can't unwittingly delete this import and still have the code "work"... but super slowly